components/item-list/item-list.jsx

```diff
@@ -3,10 +3,11 @@ import PropTypes from 'prop-types';
 import ReactGA from 'react-ga';
 import LoadingNotice from '../loading-notice.jsx';
 import ProjectCardSimple from '../project-card/project-card-simple.jsx';
+import ProfileCard from '../profile-card/profile-card.jsx';
 import Utility from '../../js/utility.js';
 import pageSettings from '../../js/app-page-settings';
 
-class ProjectList extends React.Component {
+class ItemList extends React.Component {
```

Comment:
I'm okay with using ItemList... because PulseItemList doesn't seem to add extra meaning to me since the codebase name is `pulse` and any item on this app can be a Pulse Item... depends on how you see it.

Reply:
derp, my mistake, I wrote that thinking that this is in the foundation site repo. Ignore me :P